recipe/meta.yaml

```diff
@@ -25,9 +27,10 @@ requirements:
     - pyspark
```

Comment:
```suggestion
```
Since `pyspark` has been moved to an optional dep, is it safe to remove it from here?